ntfy/terminal.py

```diff
@@ -10,9 +11,10 @@ def get_tty():
 
 
 def linux_window_is_focused():
-    window_id = int(check_output(['xprop', '-root', '\t$0',
-                                  '_NET_ACTIVE_WINDOW']).split()[1], 16)
-    return int(environ['WINDOWID']) == window_id
+    xprop_cmd = shlex.split("xprop -root _NET_ACTIVE_WINDOW")
```

Comment:
``` python
shlex.split("xprop -root _NET_ACTIVE_WINDOW")
# vs
['xprop', '-root', '\t$0', '_NET_ACTIVE_WINDOW']
```
IMHO, shlex has the following advantages:
* It is more readable: less visual clutter (quotes and commas), especially if you have a long command.
* It is more easily testable: just copy paste the command and run it on your shell.

All in all I believe that using shlex is a better practice in general and I use it on my own code, but this is not my repo, so if you prefer a list, a list it shall be :)